dcat/index.html

```diff
@@ -124,18 +124,18 @@ <h2>Introduction</h2>
 
     <p>
         Other, complementary vocabularies <em title="MAY" class="rfc2119">MAY</em> be used together with DCAT to provide more detailed format-specific information.
-      For example, properties from the VoID vocabulary [[VOID]] can be used to express various statistics about a DCAT-described dataset if that dataset is in RDF format.
+      For example, properties from the VoID vocabulary [[?VOID]] can be used to express various statistics about a DCAT-described dataset if that dataset is in RDF format.
     </p>
 
     <p>
       This document does not prescribe any particular method of deploying data expressed in DCAT.
-      DCAT is applicable in many contexts including RDF accessible via SPARQL endpoints, embedded in HTML pages as RDFa, or serialized as e.g. RDF/XML, N3, Turtle, or JSON-LD.
+      DCAT is applicable in many contexts including RDF accessible via SPARQL endpoints, embedded in HTML pages as [[?HTML-RDFa]], or serialized as e.g. RDF/XML [[?RDF-SYNTAX-GRAMMAR]], [[?N3]], [[?Turtle]], or [[?JSON-LD]].
```

Comment:
Hmm. Again - I'd expect these to be normative

Reply:
I tried to make them as normative, but then I got an error from ReSpec saying normative refs cannot be in informative sections.

Reply:
Ah - OK. Yes, I guess that makes sense.

Reply:
OK - I added a little material on DCAT 2014 to a normative section.